installer/common/shared.sh

```diff
@@ -225,7 +225,13 @@ helm upgrade $EFS_SERVER_NAME stable/$EFS_CHART_NAME \
 --install \
 --namespace "$1" \
 --timeout 600 \
+--set efsProvisioner.awsRegion="$3" \
 --set efsProvisioner.dnsName="$2.efs.$3.amazonaws.com" \
 --set efsProvisioner.efsFileSystemId="$2" \
---set efsProvisioner.awsRegion="$3"
+--set efsProvisioner.path="/" \
+--set efsProvisioner.provisionerName="kubernetes.io/aws-efs" \
+--set efsProvisioner.storageClass.name="pv-volume" \
+--set rbac.create="true" \
+--set podSecurityPolicy.enabled="false" \
+--set serviceAccount.create="false"
```

Comment:
Do we need to pass a service account name here?https://github.com/helm/charts/blob/b3bc4ff5a07e0c0a6b880b0a067df574c09848c2/stable/efs-provisioner/values.yaml#L65

Reply:
Hmmm I don't think so since we are explicitly avoiding the creating of it.

Reply:
Right now it uses the default one which might be arbitrary and cause issues.

Reply:
The default account exists per namespace and the pod for the provisioner will be launched with that account you can verify that by just looking at the token used when you do `kubectl get pod ...`.

Reply:
ahh i see what you mean, fixed